lib/rewrite.js

```diff
@@ -293,15 +288,12 @@ exports.search = function(js, searchRule) {
   // remove if one exists as the first line
   js = helpers.removeShebang(js);
 
-  var pattern = unwrapNode(esprima.parse(searchRule, {
+  var pattern = unwrapNode(parser.parse(searchRule, {
     raw: true
   }));
 
   var matches = [];
-  falafel(js, {
-    raw: true,
-    loc: true
```

Comment:
Do we know which features would be reliant on loc?

escodegen might use it to layout the code correctly?